meeteval/io/stm.py

```diff
@@ -143,3 +144,63 @@ def merged_transcripts(self) -> str:
 
     def segments(self):
         return [l.segment_dict() for l in self]
+
+    def filenames(self):
+        return {l.filename for l in self}
+
+
+def iter_examples(reference: 'STM', hypothesis: 'STM', *, allowed_empty_examples_ratio=0.1):
```

Comment:
Add a comment, that this is for internal usage?
I think we have too many functions, where the intention is internal usage, but it is not obvious.

Reply:
Yeah, we also have some functions with underscores and some without. This is currently really inconsistent